### DISCUSSION TITLE
Predicted traffic and planning output from LAV

### DISCUSSION THREAD

**Discussion by tjzetty** [dotchen/LAV#24] [Q&A]
I saw in your demo video that you had a live feed of the predicted traffic and planning. Is there an easy way to enable this output or have it saved to my computer? 

Thank you for the amazing work!

**Comment by dotchen** [comment]
Thanks for liking our work! We've released the visualization codes in team_code_v2 that will log similar visualizations with map and trajectory predictions. They will be locally stored in the `wandb` folder in the directory the commands are ran, or can be optionally viewed on the wandb.ai website (which I found to be more handy). Let me know if you have more questions!

**Reply by tjzetty** [reply]
Thank you for the quick reply! This helps a lot.

**Comment by tjzetty** [comment]
Could I also request your script for creating the LAV dataset? I would like to expand upon your work in my own research.  If you did not use a script to create the dataset could you point me in the right direction for running the code in team_code_v2 with CARLA?

**Reply by dotchen** [reply]
Sure thing, I think the easiest way to train is to use the [precollected dataset](https://utexas.box.com/s/evo96v5md4r8nooma3z17kcnfjzp2wed), and the [docs](https://github.com/dotchen/LAV/blob/main/docs/TRAINING.md) here for training. To use team_code_v2 for evaluation, you can use [this command](https://github.com/dotchen/LAV#evaluation) and it should automatically work. I will need to work on releasing the data collection script since it is a still pretty messy, but in the meantime everything should work with the released dataset. Let me know if you run into any issues or have further questions, I'm happy to assist!

**Reply by tjzetty** [reply]
The training worked very well, I look forward to seeing your data collection script!
